src/stratis_cli/_errors.py

```diff
@@ -99,15 +99,15 @@ class StratisCliPartialChangeError(StratisCliUserError):
     """
 
     def __init__(self, command, changed_resources, unchanged_resources):
-        """ Initializer.
+        """Initializer.
```

Comment:
Like here is an example. Just move ```Initializer``` down a line so it's in line with those below.

Reply:
Resolved.